backend/chunker/chunker.go

```diff
@@ -1050,16 +1089,6 @@ func (f *Fs) put(ctx context.Context, in io.Reader, src fs.ObjectInfo, remote st
 	// If previous object was chunked, remove its chunks
 	f.removeOldChunks(ctx, baseRemote)
```

Comment:
You have to pass old transaction id here.
If you don't, this will only remove v1 chunks but gather v2 chunk garbage.
Integration tests won't detect this but users won't thank you.

Reply:
Garbage collection becomes harder (for a human) with new commit method and calls for changes in `Cleanup` and for custom chunker's `About` but it's out of scope for this PR.

Reply:
Currently `removeOldChunks` doesn't accept transaction id because it rereads old metadata (if any) and rescans directory for old chunks.